### PR TITLE
Variables (RNs 41039 + 41063 + 41132)

### PR DESCRIPTION
**Pull request by EdithLansens**
@GillesBara, could you please review this pull request to confirm the accurate inclusion of release notes 41039, 41063, and 41132 in the user guide? Thanks in advance!

### CODE REVIEW

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Apply_Data_Feed.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ To apply a data feed or change the data feed of a component:
 
    - A data feed can also be provided by a **feed component**. When such a component has been added to the dashboard or low-code app, the *Feeds* section is added to the available data in the *Data* pane. You can then drag an entry from this section to a component in order to link the component to the feed component.
 
+   - From DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41039-->, data can also be provided by a **variable**. [Variables](xref:Variables) are reusable data objects that persist across sessions and can be dynamically updated.
```
Comment (GillesBara) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Apply_Data_Feed.md:44]: The "persist across sessions" might give the wrong impression that variables will keep their latest value when refreshing the app. Variables hold their initial value on app load, and can be overwritten in LCA through actions. They then hold that new value "in memory", which will be lost when you refresh the app again.

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Variables/Variables.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+   - Default view: Enter the name of a view in your DataMiner System or select from the dropdown list.
+
+   - Default object manager instance:
+
+     1. Enter the name of a module in your DataMiner System or select from the dropdown list.
+
+     1. Enter the name of an instance in your DataMiner System or select from the dropdown list.
+
+   - Default text: Enter custom text. HTML text formatting is supported, except when the variable is passed to a text input or search input component.
```
Comment (GillesBara) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Variables/Variables.md:54]: I'm not sure if we need to mention HTML text formatting here. The html format has nothing to do with the text variable. It holds text, and depending where it is used, that text can be interpreted as html.

Comment (EdithLansens) [reply user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Variables/Variables.md:54]:
```suggestion
   - Default text: Enter custom text.
```

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Apply_Data_Feed.md (hunk 1 above)
Comment (EdithLansens) [reply user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Apply_Data_Feed.md:44]:
```suggestion
   - From DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41039-->, data can also be provided by a **variable**. [Variables](xref:Variables) are reusable data objects that can be dynamically updated.
```

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**